database/changesets/20201027_WorkflowOptions.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="jwarren@broadinstitute.org"
```

Comment:
I'd certainly like jwarren@broadinstitute.org to be a long-lived email but in the case that it's not, should I use the non-Broad primary email on my github account here?

Reply:
Should `workflow_options` should be on the `workload` table instead?
(It is usually shared across multiple workflows in Cromwell.)

Reply:
Or … after discussing it with Ed, maybe not.  Carry on.

Reply:
It is on both the workload table and the table of individual workflows. The table storing workloads stores the options guaranteed across that entire workload, while the table storing the individual workflow stores the options on a per-workflow basis